### PR TITLE
Minor updates to the kafka shrading sample

### PR DESCRIPTION
**Pull request by chbatey**
Made while recording the demo

### CODE REVIEW

File: akka-sample-kafka-to-sharding-scala/processor/src/main/scala/sample/sharding/kafka/UserEvents.scala [hunk 1]
```diff
@@ -4,47 +4,43 @@ import akka.Done
...
-  def init(system: ActorSystem[_], settings: ProcessorSettings): Future[ActorRef[Message]] = {
+  def init(system: ActorSystem[_], settings: ProcessorSettings): Future[ActorRef[Command]] = {
     import system.executionContext
-    messageExtractor(settings).map(messageExtractor => {
+    KafkaClusterSharding(settings.system).messageExtractorNoEnvelope(
+      timeout = 10.seconds,
+      topic = settings.topics.head,
+      entityIdExtractor = (msg: Command) => msg.userId,
+      settings = settings.kafkaConsumerSettings()
+    ).map(messageExtractor => {
```
Comment (patriknw) [inline akka-sample-kafka-to-sharding-scala/processor/src/main/scala/sample/sharding/kafka/UserEvents.scala]: if that fails it will be silently ignored?

**Review by patriknw — APPROVED** [review]
LGTM, with and additional suggestion

File: akka-sample-kafka-to-sharding-scala/processor/src/main/scala/sample/sharding/kafka/UserEvents.scala (hunk 1 above)
Comment (chbatey) [reply akka-sample-kafka-to-sharding-scala/processor/src/main/scala/sample/sharding/kafka/UserEvents.scala]: The future is used in the pipetoSelf and rethrown if it fails https://github.com/akka/akka-samples/blob/minor-kafka-updaes/akka-sample-kafka-to-sharding-scala/processor/src/main/scala/sample/sharding/kafka/Main.scala#L48

Comment (patriknw) [reply akka-sample-kafka-to-sharding-scala/processor/src/main/scala/sample/sharding/kafka/UserEvents.scala]: ok, missed that

**Pull request merged**